src/zc/table/fieldcolumn.py

```diff
@@ -61,16 +64,17 @@ def setAnnotation(self, name, value, formatter):
     def getAnnotation(self, name, formatter, default=None):
         return formatter.annotations.get(self.key + name, default)
 
-    ###### subclass customization API ######
+    # subclass customization API ######
```

Comment:
If you're removing the `#####` decoration from the front of the comment, remove it from the back as we..